docs/DeveloperGuide.md

```diff
@@ -159,6 +160,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
       
+**UC04 - User requests for help**
+
+**MSS**
+
+1. User requests for help on how to use the system.
+2. System gives a guide to help user.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. User enters wrong help command format.
+
+    * 1a1. System gives corresponding error message.
+
+      Use case ends.
+      
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
```

Comment:
I think just java 11 will work, we shouldnt set it higher since this is supposed to work on Java 11 only